tests/signal-cast-registerer.cpp

```diff
@@ -171,9 +384,9 @@ BOOST_AUTO_TEST_CASE (custom_vector_registerer)
 // Here we make sure that two instances of the same type
 // declared in two separate libraries are resolved into the
 // same typeid.
-BOOST_AUTO_TEST_CASE (typeid_issue)
+/*BOOST_AUTO_TEST_CASE (typeid_issue)
```

Comment:
This unit test can safely be removed as type checking has been changed loooong ago to exception casting rather that using typeid.